modules/core/client/views/header.client.view.html

```diff
@@ -36,17 +36,8 @@
           <span ng-bind="authentication.user.displayName"></span> <b class="caret"></b>
         </a>
         <ul class="dropdown-menu" role="menu">
-          <li ui-sref-active="active">
-            <a ui-sref="settings.profile">Edit Profile</a>
-          </li>
-          <li ui-sref-active="active">
-            <a ui-sref="settings.picture">Change Profile Picture</a>
-          </li>
-          <li ui-sref-active="active" ng-show="authentication.user.provider === 'local'">
-            <a ui-sref="settings.password">Change Password</a>
-          </li>
-          <li ui-sref-active="active">
-            <a ui-sref="settings.accounts">Manage Social Accounts</a>
+          <li ui-sref-active="active" ng-repeat="item in accountMenu.items[0].items">
```

Comment:
Can this be changed, so we don't have to rely on an index? Just set the object that you want to iterate through, to your $scope.accountMenu object.

```javascript
// in your controller
$scope.accountMenu = Menus.getMenu('account')[0];
```